src/main/java/slimeknights/tconstruct/tools/TinkerModifiers.java

```diff
@@ -28,9 +28,9 @@
   /*
    * Items
    */
-  public static final ItemObject<Item> greenSlimeCrystal = ITEMS.register("green_slime_crystal", GENERAL_PROPS);
+  /*public static final ItemObject<Item> greenSlimeCrystal = ITEMS.register("green_slime_crystal", GENERAL_PROPS);
```

Comment:
Can probably fully remove these. Should also have handling in `TConstruct` for migration.